src/components/tasks/card/index.tsx

```diff
@@ -38,8 +34,15 @@ const Card: FC<Props> = ({ content }) => {
   const iconHeight = '25px';
   const iconWidth = '25px';
 
+  const cardClassNames = [classNames.card];
+  const currentDate = new Date();
+  if (status !== COMPLETED) {
+    if (localEndsOn.valueOf() - currentDate.valueOf() <= 0) {
+      cardClassNames.push(classNames.overdueTask);
+    }
+  }
```

Comment:
Move this logic to a separate function called `isTaskOverdue(status, localEndsOn)` then use this function on line number 45 to add class.

Reply:
Made the changes. Please check

Reply:
Made the changes, please check